src/main/java/view/ResultView.java

```diff
@@ -44,30 +42,20 @@ private static void printHeader() {
         System.out.println("============");
     }
 
-    private void printCountOfPrize(Map<Integer, Integer> countOfPrize, Set<Integer> correctCounts) {
-        correctCounts.stream()
-                .filter(count -> count > 2)
-                .sorted()
-                .forEach(correctCount -> printEach(countOfPrize, correctCount));
+    private void printCountOfPrize(Map<Prize, Integer> countOfPrize, Set<Prize> prizes) {
+        prizes.stream()
+                .filter(prize -> prize != Prize.NONE)
+                .forEach(prize -> printEach(countOfPrize, prize));
     }
 
-    private void printEach(Map<Integer, Integer> countOfPrize, Integer correctCount) {
-        final Money prizeMoney = getPrizeMoney(correctCount);
-        System.out.println(correctCount + "개 일치 (" + prizeMoney.wonString() + ") - " + countOfPrize.getOrDefault(correctCount, ZERO) + "개");
+    private void printEach(Map<Prize, Integer> countOfPrize, Prize prize) {
```

Comment:
countOfPrize전체를 넘기지 말고, countOfPrize.get(prize)값만 넘기는건 어떨까요?
```suggestion
    private void printEach(Integer countOfPrize, Prize prize) {
```